aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -56,25 +56,25 @@ static inline Tensor& unary_op_impl_(Tensor& self, OutImpl& out_impl) {
 }
 
 Tensor& bitwise_not_out(Tensor& result, const Tensor& self) { return unary_op_impl_out(result, self, bitwise_not_stub); }
-Tensor bitwise_not(const Tensor& self) { return unary_op_impl(self, bitwise_not_out); }
-Tensor& bitwise_not_(Tensor& self) { return unary_op_impl_(self, bitwise_not_out); }
+Tensor bitwise_not(const Tensor& self) { return unary_op_impl(self, at::bitwise_not_out); }
```

Comment:
we need to make this less error prone.  At a minimum, you need a comment.

Reply:
Looks like we need to start adding regexp based linters.

Reply:
I think the actual way to test for this is:
1) Implement @mruberry's test suggestions so we can plug in a device to our tests
2) Have a generic way of registering "operator fallback override" for a device (i.e. unboxed->boxed support) CC @bwasti 
3) Add a fake test device
4) Fallback every function on our fake test device and very we get called

I'm not 100% sure this will work, it depends on how exactly fallback is implemented (i.e. if we only fallback functions with device-specific dispatch).